packages/ui-app/src/Row.tsx

```diff
@@ -109,14 +109,18 @@ export const styles = `
     }
   }
 
-  .ui--Row-name {
+  .ui--Row-address-or-name {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
     overflow: hidden;
     text-overflow: ellipsis;
     text-transform: uppercase;
     white-space: normal;
+
+    .withAddr {
+      text-transform: lowercase;
```

Comment:
This is not correct - addresses have uppercase letters as well.

Reply:
For AddressMini, this is what was done - https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/AddressMini.tsx#L137

Apply the text-transform to `withName` only, so `withAddr` has just the default.